tests/gatsby-starter-default/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 version: '3'
 services:
   api:
-    image: "kitconcept/plone.restapi"
+    image: "kitconcept/plone.restapi:2.2.1"
```

Comment:
@sneridagh The latest kitconcept/plone.restapi (3.x) had disallowed adding Collections. I'm curious why? Any change to have an optional version of the image where collections are allowed? (It's hard to enable them programmatically at CI.)